x-pack/plugins/security_solution/public/common/containers/alerts/use_alert_prevalence_from_process_tree.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useEffect, useState } from 'react';
+
+import { useQuery } from 'react-query';
+import { useHttp } from '../../lib/kibana';
+
+// import { DEFAULT_MAX_TABLE_QUERY_SIZE } from '../../../../common/constants';
```

Comment:
nit: these may be deleted

Reply:
update: if this is commented-out example code for when the `/TBD` endpoint exists, please ignore this comment

Reply:
Yep, that is what this was for 👍